sources/packages/backend/workflow/src/workflow-definitions/assessment-gateway.bpmn

```diff
@@ -789,10 +568,342 @@
+    <bpmn:sequenceFlow id="Flow_1mbspca" sourceRef="student-income-verification-subprocess" targetRef="Gateway_0jc5bz6" />
+    <bpmn:sequenceFlow id="Flow_1uh1kyu" sourceRef="Gateway_0ny5tkb" targetRef="Gateway_0jc5bz6" />
+    <bpmn:sequenceFlow id="Flow_1rn6agw" sourceRef="Gateway_1jdrla2" targetRef="Gateway_0jc5bz6" />
+    <bpmn:sequenceFlow id="pir-pending-declined-flow" name="Declined" sourceRef="pir-pending-declined-complted-gateway" targetRef="pir-declined-end-event">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=programInfoStatus = "Declined"</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="pir-pending-completed-flow" name="Completed" sourceRef="pir-pending-declined-complted-gateway" targetRef="Gateway_1hgcddi">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=programInfoStatus = "Completed"</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="Flow_06144it" sourceRef="program-info-not-required-task" targetRef="Gateway_1hgcddi" />
+    <bpmn:sequenceFlow id="Flow_0im6s57" sourceRef="Gateway_03gbgqa" targetRef="pir-pending-declined-complted-gateway" />
+    <bpmn:sequenceFlow id="Flow_0cxn3c5" sourceRef="program-info-request-completed-message" targetRef="Gateway_03gbgqa" />
+    <bpmn:sequenceFlow id="pir-completed-flow" name="Yes" sourceRef="pir-pending-gateway" targetRef="Gateway_03gbgqa">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=programInfoStatus != "Required"</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="Flow_0joyxdn" sourceRef="pir-pending-event-gateway" targetRef="program-info-request-completed-message" />
+    <bpmn:sequenceFlow id="pir-not-completed-flow" name="No" sourceRef="pir-pending-gateway" targetRef="pir-pending-event-gateway">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=programInfoStatus = "Required"</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="Flow_1ds5r8m" sourceRef="pir-pending-event-gateway" targetRef="program-info-request-timer" />
+    <bpmn:sequenceFlow id="Flow_1sze0w6" sourceRef="program-info-request-timer" targetRef="program-info-required-task" />
+    <bpmn:sequenceFlow id="Flow_0guy66a" sourceRef="program-info-required-task" targetRef="pir-pending-gateway" />
+    <bpmn:sequenceFlow id="needs-pir-flow" name="Needs PIR" sourceRef="needs-pir-gateway" targetRef="program-info-required-task">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=studentDataSelectedOffering = null</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="pir-not-required-flow" name="PIR not required or Change in progress" sourceRef="needs-pir-gateway" targetRef="program-info-not-required-task">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=studentDataSelectedOffering != null or applicationEditStatus = "Change in progress"</bpmn:conditionExpression>
```

Comment:
I feel that it will be good to use a separate gateway for the deviation based on edit status. 
![image](https://github.com/user-attachments/assets/0a81b3b1-9df3-4492-b7ce-208ece8ad0f9)

Reply:
Same for PIR. 
![image](https://github.com/user-attachments/assets/ff72a432-bdc1-4edf-9340-03311776a6cc)

Reply:
Lets check PIR first, fo you mean something like below?

![image](https://github.com/user-attachments/assets/7a276a17-5e0f-427b-9b7e-6c28de924cab)

Instead of the "OR" condition as below?

![image](https://github.com/user-attachments/assets/67c114fb-2bce-4153-b1e4-b75e2b4f77bc)

Reply:
I see -  The new path also requires program info status to be updated. I thought it will skip the whole PIR block.

Reply:
As per our conversation, that is how it will look like.
![image](https://github.com/user-attachments/assets/557a674b-1f32-4479-8acb-da8322261a0d)